### PR TITLE
translation of faq.md

### PR DESCRIPTION
**Pull request by EricKani**
## Motivation

Translation of docs/faq.md

## Modification

docs_zh_CN/faq.md

### CODE REVIEW

File: docs_zh_CN/faq.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
 ## 常见问题
 
-欢迎有兴趣的朋友一起翻译 MMCV 文档。如有兴趣，请在 [MMCV issue](https://github.com/open-mmlab/mmcv/issues) 提 issue 确定翻译的文档。
+我们在这里列出了一些使用者遇到的共通的问题以及他们对应的解决方法。如果您遇到了任何其他常见的问题，并且知道可以帮到大家的解决办法，
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:3]:
```suggestion
在这里我们列出了用户经常遇到的问题以及对应的解决方法。如果您遇到了其他常见的问题，并且知道可以帮到大家的解决办法，
```

File: docs_zh_CN/faq.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+欢迎随时丰富这个列表。
+
+- MMCV和MMDetection的兼容性问题；"ConvWS is already registered in conv layer"
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:6]:
```suggestion
- MMCV 和 MMDetection 的兼容性问题；"ConvWS is already registered in conv layer"
```

File: docs_zh_CN/faq.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+    
+    请按照上述说明为您的 MMDetection 版本安装正确版本的 MMCV。
+    
+- "No module named 'mmcv.ops'"; "No module named 'mmcv._ext'"。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:10]:
```suggestion
- "No module named 'mmcv.ops'"; "No module named 'mmcv._ext'"
```

File: docs_zh_CN/faq.md [hunk 4]
```diff
@@ -1,3 +1,37 @@
...
+
+    1. 使用`pip uninstall mmcv`卸载您环境中的mmcv。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:12]:
```suggestion
    1. 使用`pip uninstall mmcv`卸载您环境中的 mmcv
```

File: docs_zh_CN/faq.md [hunk 5]
```diff
@@ -1,3 +1,37 @@
...
+    2. 按照上述说明安装mmcv-full。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:13]:
```suggestion
    2. 按照上述说明安装 mmcv-full
```

File: docs_zh_CN/faq.md [hunk 6]
```diff
@@ -1,3 +1,37 @@
...
+
+- "invalid device function" 或者 "no kernel image is available for execution"。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:15]:
```suggestion
- "invalid device function" 或者 "no kernel image is available for execution"
```

File: docs_zh_CN/faq.md [hunk 7]
```diff
@@ -1,3 +1,37 @@
...
+
+    1. 检查您的GPU的CUDA计算能力。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:17]:
```suggestion
    1. 检查 GPU 的 CUDA 计算能力
```

File: docs_zh_CN/faq.md [hunk 8]
```diff
@@ -1,3 +1,37 @@
...
+    2. 运行  `python mmdet/utils/collect_env.py` 来检查是否Pytorch、torchvision和MMCV是否是针对正确的GPU架构构建的。
+        您可能需要去设置 `TORCH_CUDA_ARCH_LIST` 来重新安装MMCV。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:19]:
```suggestion
        您可能需要去设置 `TORCH_CUDA_ARCH_LIST` 并重新安装 MMCV
```

File: docs_zh_CN/faq.md [hunk 9]
```diff
@@ -1,3 +1,37 @@
...
+        在使用旧版的GPUS时，如：colab上的Tesla K80 (3.7)，是可能发生兼容性问题的。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:20]:
```suggestion
        兼容性问题的可能会出现在使用旧版的 GPUs，如：colab 上的 Tesla K80 (3.7)
```

File: docs_zh_CN/faq.md [hunk 10]
```diff
@@ -1,3 +1,37 @@
...
+    3. 检查运行环境是否和mmcv/mmdet编译时的环境相同。例如，您可能使用CUDA 10.0编译的mmcv，但是在CUDA 9.0的环境中运行它。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:21]:
```suggestion
    3. 检查运行环境是否和 mmcv/mmde t编译时的环境相同。例如，您可能使用 CUDA 10.0 编译 mmcv，但在 CUDA 9.0 的环境中运行它
```

File: docs_zh_CN/faq.md [hunk 11]
```diff
@@ -1,3 +1,37 @@
...
+
+- "undefined symbol" 或者 "cannot open xxx.so"。
+    
+    1. 如果那些符号是CUDA/C++的（例如：libcudart.so 或者 GLIBCXX），检查CUDA/GCC运行时是否和编译mmcv时使用的一样。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:25]:
```suggestion
    1. 如果符号和 CUDA/C++ 相关（例如：libcudart.so 或者 GLIBCXX），请检查 CUDA/GCC 运行时的版本是否和编译 mmcv 的一致
```

File: docs_zh_CN/faq.md [hunk 12]
```diff
@@ -1,3 +1,37 @@
...
+    2. 如果那些符号是Pytorch符号（例如：符号包含caffe、aten和TH），检查Pytorch的版本是否和编译mmcv时使用的一样。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:26]:
```suggestion
    2. 如果符号和 PyTorch 相关（例如：符号包含 caffe、aten 和 TH），请检查 PyTorch 运行时的版本是否和编译 mmcv 的一致
```

File: docs_zh_CN/faq.md [hunk 13]
```diff
@@ -1,3 +1,37 @@
...
+    3. 运行 `python mmdet/utils/collect_env.py` 以检查 PyTorch、torchvision 和 MMCV 构建和运行的环境是否相同。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:27]:
```suggestion
    3. 运行 `python mmdet/utils/collect_env.py` 以检查 PyTorch、torchvision 和 MMCV 构建和运行的环境是否相同
```

File: docs_zh_CN/faq.md [hunk 14]
```diff
@@ -1,3 +1,37 @@
...
+
+- "RuntimeError: CUDA error: invalid configuration argument"。
+
+    这个错误可能是由于您的GPU性能不佳造成的。尝试降低[THREADS_PER_BLOCK](https://github.com/open-mmlab/mmcv/blob/cac22f8cf5a904477e3b5461b1cc36856c2793da/mmcv/ops/csrc/common_cuda_helper.hpp#L10)
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:31]:
```suggestion
    这个错误可能是由于您的 GPU 性能不佳造成的。尝试降低[THREADS_PER_BLOCK](https://github.com/open-mmlab/mmcv/blob/cac22f8cf5a904477e3b5461b1cc36856c2793da/mmcv/ops/csrc/common_cuda_helper.hpp#L10)
```

File: docs_zh_CN/faq.md [hunk 15]
```diff
@@ -1,3 +1,37 @@
...
+    的值并重新编译mmcv。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:32]:
```suggestion
    的值并重新编译 mmcv。
```

File: docs_zh_CN/faq.md [hunk 16]
```diff
@@ -1,3 +1,37 @@
...
+
+- "RuntimeError: nms is not compiled with GPU support"。
+    
+    这个错误是由于您的CUDA环境没有正确安装。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:36]:
```suggestion
    这个错误是由于您的 CUDA 环境没有正确安装。
```

File: docs_zh_CN/faq.md [hunk 17]
```diff
@@ -1,3 +1,37 @@
...
+    您可以尝试重新安装您的CUDA环境，然后在重新编译mmcv前删除build/文件夹。
```
Comment (zhouzaida) [inline docs_zh_CN/faq.md:37]:
```suggestion
    您可以尝试重新安装您的 CUDA 环境，然后删除 mmcv/build 文件夹并重新编译 mmcv。
```

**Review by ZwwWayne — APPROVED** [review]

**Comment by OpenMMLab-Coodinator** [comment]
Hi @EricKani ！First of all, we want to express our gratitude for your significant PR in this project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**